standard/icon_data.lua

```diff
@@ -0,0 +1,10 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Icon/Data
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+
+return {
+	check = 'fas fa-check',
```

Comment:
```suggestion
	check = 'fas fa-check',
	cross = 'fas fa-times',
	questionMark = 'fas fa-question',
	line = 'fas fa-minus',
```

Reply:
hmm i suggested those since iirc those are the ones we use on the git for now
(am on phone atm)

Reply:
There's plenty more, those are the ones used by SC though